guides/common/modules/con_mirroring-policies-overview.adoc

```diff
@@ -21,3 +21,6 @@ Complete Mirroring::
 Mirrors content as well as repodata.
 This is the fastest method.
 This mirroring policy is only available for Yum content.
+Republishing metadata in repositories with Complete Mirroring is high-risk.
```

Comment:
Does that even make sense? If you republish the metadata, it's hardly "complete mirroring" anymore. What happens if metadata is signed, for example with SUSE content?

_If_ we really want to document this, I'd _at least_ add a "WARNING" admonition or say that "{Team} does not recommend to republish metadata for repositories with 'complete mirroring'" or similar.

Reply:
That is right. Once you republish metadata locally, that repo is no longer technically Mirror Complete till the next complete sync of the repo which will overwrite the locally generated metadata with the upstream metadata again. The warning is trying to communicate that with the 'high-risk' warning. We don't have the option to do this on the UI (to make this harder to do). Only hammer allows this with a --force option for some corner cases.

Reply:
I can add a "WARNING" to this. I initially thought about doing that but wanted to see what reviewers might think.